### PR TITLE
[RFC] build: update dependencies for busted

### PR DESCRIPTION
**Pull request by jszakmeister**
We didn't have ansicolors pegged, and several others were below the
minimum required versions causing busted to go out and grab newer
versions anyways.  Let's peg them all to useful versions.

### CODE REVIEW

File: third-party/cmake/BuildLuarocks.cmake [hunk 1]
```diff
@@ -69,17 +69,19 @@ if(USE_BUNDLED_BUSTED)
     COMMAND ${LUAROCKS_BINARY}
     ARGS build say 1.3-0 ${LUAROCKS_BUILDARGS}
     COMMAND ${LUAROCKS_BINARY}
-    ARGS build luassert 1.7.4-0 ${LUAROCKS_BUILDARGS}
+    ARGS build luassert 1.7.6-0 ${LUAROCKS_BUILDARGS}
     COMMAND ${LUAROCKS_BINARY}
     ARGS build lua-term 0.1-1 ${LUAROCKS_BUILDARGS}
```
Comment (fwalch) [inline third-party/cmake/BuildLuarocks.cmake]: Current version of lua-term is 0.3-1, don't know if we want to update that.

Comment (jszakmeister) [inline third-party/cmake/BuildLuarocks.cmake]: I went ahead and updated them.  lua-term has been at 0.3-1 for quite some time, so it seems like a small oversight that we were pegged at an older one.  The dkjson packaging update seemed only geared towards supporting Lua 5.3 but didn't appear to affect us, but I updated it too anyways.

**Comment by fwalch** [comment]
LGTM.

**Comment by justinmk** [comment]
LGTM

**Comment by jszakmeister** [comment]
Thanks!

**Pull request merged**